six/modules/c++/six.sidd/source/GeoTIFFWriteControl.cpp

```diff
@@ -39,7 +39,7 @@ GeoTIFFWriteControl::GeoTIFFWriteControl()
                                                     tiff::Const::Type::ASCII,
                                                     Constants::GT_XML_TAG);
 }
-void GeoTIFFWriteControl::initialize(Container* container)
+void GeoTIFFWriteControl::initialize(mem::SharedPtr<Container> container)
```

Comment:
Do this under a separate pull request but can you add unittests showing reading and writing SIDD GeoTIFFs works if there aren't some already?